### PR TITLE
Creates a Search Bar that opens by default at the top of the map.

### PR DESCRIPTION
**Pull request by marishuman**
## What?
This creates a search bar that opens by default at the top of the map. The search bar displays a list of autocompleted addresses as the user types an address in. Once the user inputs an address, the map zooms into the address' location and pinpoints the address.
## Why? 
The search bar is useful for users wanting to know if they are covered by Seattle Community Network. By pinpointing their address, users can tell if their address lies within one of the Organization's coverage areas.
## How? 
Uses the leaflet-geosearch library to implement the search bar.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "qrcode.react": "^1.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "leaflet-geosearch": "^3.7.0",
```
Comment (ysono) [inline package.json:34]: Let's sort dependencies alphabetically.

File: src/vis/MeasurementMap.tsx [hunk 2]
```diff
@@ -126,6 +128,12 @@ const MeasurementMap = ({
       setSLayer(L.layerGroup().addTo(_map));
       setMLayer(L.layerGroup().addTo(_map));
       setLLayer(L.layerGroup().addTo(_map));
+
+      const search = new (GeoSearchControl as any)({
+        provider: new OpenStreetMapProvider(),
+        style: 'bar', // optional: bar|button  - default button
```
Comment (ysono) [inline src/vis/MeasurementMap.tsx:134]: The comment came from the library's documentation right? We can remove them here. Readers of this code can always look it up.

**Comment by ysono** [comment]
Well done! With some comments, approved. :shipit:

**Comment by infrared0** [comment]
Thanks Mariana! And thanks for the review Yoshi!

**Pull request merged**